recipes-core/tdx-enc-handler/tdx-enc-handler/tdx-enc-caam.sh

```diff
@@ -61,6 +70,31 @@ tdx_enc_key_gen() {
     fi
 }
 
+# backup original data in partition (if not encrypted)
+tdx_enc_backup_data() {
+    if [ ${TDX_ENC_PRESERVE_DATA} -eq 1 ]; then
+        mkdir -p "${TDX_ENC_STORAGE_MOUNTPOINT}"
+        if mount ${TDX_ENC_STORAGE_LOCATION} "${TDX_ENC_STORAGE_MOUNTPOINT}"; then
+            tdx_enc_log "Backing up original content..."
```

Comment:
I think the `mkdir` line is still needed or the mount command would fail if the directory doesn't already exist (as is the case in my image).
I will apply this change and test this morning.